lacework/data_source_lacework_metrics.go

```diff
@@ -0,0 +1,60 @@
+package lacework
+
+import (
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/lacework/go-sdk/api"
+)
+
+func dataSourceLaceworkMetrics() *schema.Resource {
+	return &schema.Resource{
+		Read: dataLaceworkMetricsRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The name of the module",
+			},
+			"version": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The version of the module",
+			},
+			"dataset": {
```

Comment:
I would like users to modify this value, so let's not give them that control.

Reply:
Fixed!